kythe/go/extractors/config/runextractor/README.md

```diff
@@ -32,6 +32,7 @@ Because `runextractor` is invoked in a manner not conducive to cleanly passing
 commandline flags, there is non-trivial setup done with environment variables.
 When calling `runextractor`, here are the relevant environment variables:
 
+* **KYTHE_CORPUS**: The corpus set for extracted files.
```

Comment:
Can you please also mention which of these are "allowed" vs. "required" for things to work?

And maybe give some examples or other context?

Reply:
elsewhere we've mostly used the phrase "corpus label" as the term of art for this—the corpus is the group of files, the corpus label is its identifier.